dvc/remote/local.py

```diff
@@ -144,8 +143,9 @@ def isdir(path_info):
     def getsize(path_info):
         return os.path.getsize(fspath_py35(path_info))
 
-    def walk(self, path_info):
-        return dvc_walk(path_info, self.repo.dvcignore)
+    def walk_files(self, path_info):
+        for fname in walk_files(str(path_info), self.repo.dvcignore):
+            yield path_info / os.path.relpath(fname, path_info)
```

Comment:
`walk_files` returns abs paths, so
```suggestion
            yield PathInfo(fname)
```
should be enough